include/Flow.h

```diff
@@ -0,0 +1,90 @@
+/*
+ * Content: reads water flow sensor and creates packets, returns string and package
+ * Author : Ryan Leung
+ * Team   : Sustaingineering UBC
+ */
+ 
+#pragma once
+#include <Arduino.h>
+#include "LoRaTransceiver.h"
+#include "farmSensor.h"
+ 
+enum sensorType { digital, analog, counter };
+ 
+class flow : public farmSensor{
+ 
+private:
+    int m_pin;
```

Comment:
No need to redeclare a private or public member that already declared in the inherited class unless you want to override it.

Reply:
Remember to add the keyword `virtual` to the members you override in farmsensor.h and add the keyword `override` in flow.h